src/java/org/apache/cassandra/tools/nodetool/GetDefaultKeyspaceRF.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I think it makes sense to write unit tests for the new nodetool commands (similar to `GossipInfoTest`). Even though the logic is trivial, having tests helps to keep code coverage on a high level and ensures nothing is broken occasionally.